nntrainer/models/neuralnet.cpp

```diff
@@ -393,17 +393,8 @@ sharedConstTensors NeuralNetwork::forwarding(
         unsigned int lookahead =
           std::get<props::MemorySwapLookahead>(model_flex_props);
 
-        if (lookahead != 0) {
-          if ((f) % (lookahead + 1) == lookahead - 1) {
-            std::cout << "request load tensor : " << f + lookahead + 1
-                      << std::endl;
-            ml_logd("request load tensor for %d", f + 1);
-            model_graph.LoadTensors((f / (lookahead + 1) + 1) *
-                                    (lookahead + 1));
-          }
-        } else {
-          model_graph.LoadTensors(f);
-        }
+        if (f != 0 && lookahead != 0)
+            model_graph.LoadTensors(f);
```

Comment:
for lookahead = 0 case (on demand) 
doesn't we need this code ? 
```.cpp
else {
          model_graph.LoadTensors(f);
        }
```

Reply:
Removed the condition Lookahead != 0.